fr/appel-contributions.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: blank
+title: Appel à rédiger un tutoriel
+---
+
+# Tutoriels en cours de préparation 
+
+[Vous pouvez consulter la liste de tous les tutoriels en cours de préparation, toutes langues confondues, dans notre dépôt ph-submissions.](https://github.com/programminghistorian/ph-submissions/issues)
+
+# Appel à rédiger un tutoriel 
+
+Le comité éditorial de la version francophone du _Programming Historian_ est particulièrement intéressé à recevoir des propositions de tutoriels sur les sujets suivants. Merci de consulter aussi, pour plus d'informations, nos [consignes aux auteur(e)s](/fr/consignes-auteurs):
+
+1. How do you conduct spatial clustering of geographic data?
+
+    We’ve got a great set of introductory mapping lessons, and while they are great for teaching how to make nice visualisations, we’ve not yet branched deeply enough into more advanced analysis skills. One of the most useful is the application of clustering algorithms, which identify logical groups of individual points in geographic space. Useful for forming conclusions on anything from trade to migration. But like with all analyses, it’s a space (no pun intended) fraught with pitfalls for the uninitiated. We’re looking for a great introduction that highlights both the strengths and the challenges of this form of analysis.
```

Comment:
Is this still waiting to be translated? Or is English text here accidentally?

Reply:
Had left this for the end bcz had trouble with translating some terms. Done now:)